### DISCUSSION TITLE
How to save figure without showing it?

### DISCUSSION THREAD

**Discussion by imirzayev** [alandefreitas/matplotplusplus#392] [Q&A]
I want to save the plots in the image without showing them within a loop. However, the plots still pop up even if I do not use show() function.

**Comment by Kabourlix** [comment]
You should check in Coding Style section of the website [here](https://alandefreitas.github.io/matplotplusplus/coding-styles/reactive-vs-quiet-figures/) which provides the solution to you're issue.

**Comment by jpeter42** [comment]
The example in the link does not seem to work:

auto f = figure(true);
auto ax = f->gca();

yields `class matplot::figure_type'} has no member named `gca'
